statsmodels/stats/tests/test_lilliefors.py

```diff
@@ -0,0 +1,26 @@
+from numpy.testing import assert_, assert_raises
+from scipy import stats
+import nose
+from statsmodels.stats._lilliefors import lilliefors
+import numpy as np
+
+class TestLilliefors(object):
```

Comment:
The unit tests are not very strict. Are there some reference numbers from other packages available.